<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorTemplateWidget</class>
 <widget class="QWidget" name="InspectorTemplateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>483</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>483</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFlashlayer">
     <item>
      <spacer name="horizontalSpacerFlashlayer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>34</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelFlashlayer">
       <property name="text">
        <string>&lt;a href=&quot;Flash layer&quot; style=&quot;color: #BEBEBE;&quot;&gt;Flash layer&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxFlashlayer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInvoke">
     <item>
      <spacer name="horizontalSpacerInvoke">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>55</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelInvoke">
       <property name="text">
        <string>&lt;a href=&quot;Invoke&quot; style=&quot;color: #BEBEBE;&quot;&gt;Invoke&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditInvoke">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutLoop">
     <item>
      <spacer name="horizontalSpacerLoop1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelUseStoredData">
       <property name="text">
        <string>&lt;a href=&quot;Use stored data&quot; style=&quot;color: #BEBEBE;&quot;&gt;Use stored data&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxUseStoredData">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerLoop2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutData">
     <item>
      <widget class="QTableWidget" name="tableWidgetTemplateData">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>21</number>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutRow">
     <item>
      <widget class="QToolButton" name="toolButtonAddRow">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Add new device</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRemoveRow">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Remove device</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRow">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerTemplate">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxFlashlayer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>flashlayerChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonAddRow</sender>
   <signal>clicked()</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>addRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRemoveRow</sender>
   <signal>clicked()</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>removeRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidgetTemplateData</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>itemChanged(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditInvoke</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>invokeChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelFlashlayer</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>resetFlashlayer(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelInvoke</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>resetInvoke(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseStoredData</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>useStoredDataChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelUseStoredData</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorTemplateWidget</receiver>
   <slot>resetUseStoredData(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>flashlayerChanged(int)</slot>
  <slot>addRow()</slot>
  <slot>removeRow()</slot>
  <slot>itemChanged(QTableWidgetItem*)</slot>
  <slot>invokeChanged(QString)</slot>
  <slot>resetFlashlayer(QString)</slot>
  <slot>resetInvoke(QString)</slot>
  <slot>useStoredDataChanged(int)</slot>
  <slot>resetUseStoredData(QString)</slot>
 </slots>
</ui>
